src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java

```diff
@@ -388,6 +388,9 @@ public Object createValue(UIDefaults defaultsTable) {
         final Color focusRingColor = AquaImageFactory.getFocusRingColorUIResource();
         final Border focusCellHighlightBorder = new BorderUIResource.LineBorderUIResource(focusRingColor);
 
+        // for table cell highlighter
+        final Color cellFocusRing = AquaImageFactory.getCellHighlightColorUIResource();
```

Comment:
overall looks ok. I believe you have tested well in BigSur and Monterey also..
Minor nit, this cellFocusRing should be "cellFocusRingColor" like other color variables.

Reply:
@prsadhuk Tested it on BigSur and Monterey, it works as expected. I will change the variable name as suggested and updating the PR.

Reply:
@prsadhuk Updated the PR. A note on the test case, since the test case is automated it tests the cell focus ring with colors loaded during the start of program execution and any on-the-fly accent color changes were tested manually.